crates/turborepo-lib/src/daemon/server.rs

```diff
@@ -547,10 +547,13 @@ impl proto::turbod_server::Turbod for TurboGrpcServiceInner {
         tokio::spawn(async move {
             loop {
                 let event = match package_changes_rx.recv().await {
-                    Err(err) => {
-                        error!("package changes stream closed: {}", err);
-                        break;
-                    }
+                    Err(err) => proto::PackageChangeEvent {
```

Comment:
Should we close the stream here? What's the intended behavior when a client receives an error?